webpack/assets/javascripts/react_app/common/rtlTestHelpers.js

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { i18nProviderWrapperFactory } from './i18nProviderWrapperFactory';
+
+const mockStore = configureMockStore([thunk]);
```

Comment:
I thought we wanted to use a real store instead?

Reply:
I didn't want to inflate the PR. Let's keep this refactoring to a separate PR (and probably RFE)